DQM/TrackingMonitor/src/TrackAnalyzer.cc

```diff
@@ -517,6 +522,42 @@ void TrackAnalyzer::bookHistosForHitProperties(DQMStore::IBooker & ibooker) {
 	algorithm->setBinLabel(ibin+1,reco::TrackBase::algoNames[ibin]);
         oriAlgo->setBinLabel(ibin+1,reco::TrackBase::algoNames[ibin]);
       }
+
+      // DataFormats/TrackReco/interface/TrajectoryStopReasons.h
+      std::vector<std::string> StopReasonName;
```

Comment:
@mtosi - pls change to

std::vector<std::string> stopReasonName={"UNINITIALIZED","MAX_HITS"....};

thx.